recipients_app/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -61,7 +61,7 @@
       </BuildableProductRunnable>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Profile"
+      buildConfiguration = "Release-prod"
```

Comment:
shouldnt this be Profile-Prod?

Reply:
It should, thanks for catching 🙇 [b39c381](https://github.com/socialincome-san/public/pull/273/commits/b39c3819343ebd45e065a53323d1e0a415126488)